### ISSUE TITLE
Navigation

### ISSUE THREAD

**Issue by cfreak** [edalzell/statamic-blade#4]
Hello,

first of all i like your Plugin. :-)

Is it possible to get the Navigation with your Plugin?

Or do you have any ideas or plans how to Parse the Navigation of Statamic V3 in Blade Templates?

Best Regards
Max

**Comment by edalzell** [comment]
Do you mean the `nav` tag?

**Comment by phpsa** [comment]
Yes, i do believe that is what is wanted there. -- or the ability to call tags would be good to.

**Comment by edalzell** [comment]
@phpsa can you please give an example of what you'd like for the @nav directive?

Also, you say "call tags", can you please be more specific and open an FR for the ones that are the highest priority for you?

**Comment by addgod** [comment]
It would be nice with a @nav directive. Unless there is a way to use on of the other tags to get em.

**Comment by edalzell** [comment]
Progress:
```
@nav('footer')
    @foreach($nav as $item)
        {{ $item['title'] }}
    @endforeach
@endnav
```

**Comment by edalzell** [comment]
Released in 1.3
